lib/factorial.dart

```diff
@@ -0,0 +1,16 @@
+// Define a function.
+int computeFactorial(int aNumber) {
+  var factorial = 1;
+  for (var i = 1; i <= aNumber; i++) {
+    factorial = factorial * i;
+  }
+  return factorial;
+  //print('The factorial of $aNumber is $factorial'); // Print to console.
```

Comment:
What is the objetive of this comment?

Reply:
First or second? The first comment is from the tutorial and the second I left for remembering in case I have to print the output.